docs/Documentation/Scripting/Building-Blocks/Integration-List.md

```diff
@@ -530,6 +530,32 @@ The syntax is as follows:
 Check the [Luck Perms documentation](https://luckperms.net/wiki/Context)
 for an in-depth explanation on what contexts are and how to add them to permissions.
 
+### Permissions
+In case you still want to add and remove permissions directly and don't want to have the LuckPerms changelog chat spam 
+when executing the LuckPerms command via the Server or a player with enough permissions you can use the `luckperms 
+addPermission` and `luckperms removePermission` events.
```

Comment:
```suggestion
If you prefer to directly add or remove permissions without triggering the LuckPerms changelog chat notifications,
you can utilize the `luckperms addPermission` and `luckperms removePermission` events. 
```
I think this would be less confusing